### DISCUSSION TITLE
Use Dummy Plug instead of a virtual display?

### DISCUSSION THREAD

**Discussion by lycantrophic** [ClassicOldSong/Apollo#121] [Ideas]
First of all, I want to thank you for all your amazing work. Your project almost has all the enhancements that I could ever want, it is a joy to use. 

On to the topic, sorry if it can already be done; I'm using Apollo for 3 seperate devices. While virtual displays are convenient; colors are washed out compared to physical output and can't access nvidia control panel. Thinking I could get better results with a dummy plug I ordered one. 

My question (or suggestion if not present) is can we use dummy display instead of virtual? I'd like to be able to do all the things virtual display does (switch displays, change resolutions etc.). Thank you.

**Comment by ClassicOldSong** [comment]
It can't change to any arbitrary resolution like using virtual display. I don't have the color washed out issue, maybe you have enabled HDR on an unsupported device by accident?

**Reply by lycantrophic** [reply]
I double checked. HDR is closed on settings, never enable it. 

What I want is something different, virtual displays work with limited output display range whereas physical and dummy can show full output display range, so better brightness color contrast overall.

**Reply by ClassicOldSong** [reply]
Tbh I couldn't notice any difference streaming from my physical display than from virtual display. I have physical display cinfigered to full color range.

Could you provide a screenshot comparison of the color difference streaming from your physical display/dummy plug/virtual display? It's better if you could provide the edid dump from your monitor and dummy plug.

**Comment by lycantrophic** [comment]
![Screenshot_2024-11-29-10-10-47-462_com limelight noir](https://github.com/user-attachments/assets/5bef47e8-ecd6-4a97-ab58-a609cea2b1e4)
Sorry for the late reply, was waiting for the dummy plug to arrive. The upper one is from the dummy plug, the bottom is the virtual display. While the images are dark, difference in yellow is obvious imo. 

Also I uploaded the image to chatgpt to make sure i wasn't imagining things, it gave me this:

There is a measurable difference in the dynamic range of yellow tones between the top and bottom images:

"Top Image Yellow Average (RGB): [137.60, 122.96, 79.48]
Bottom Image Yellow Average (RGB): [126.90, 117.57, 82.57]
The top image has slightly higher red and green values, making the yellows appear a bit brighter and more vivid, while the bottom image has a slightly lower dynamic range in those tones. Conversely, the blue channel is marginally stronger in the bottom image, which may contribute to a slightly muted yellow perception. ​"

I can test with images from other games if needed. Thank you.

**Reply by ClassicOldSong** [reply]
Would you try with some monitor test programs? For example the online one provided by EIZO?

Testing in-game image can be affected by many different factors like the time of animation/viewing angle, and ChatGPT can be hallucinating.
